test/E2E/Browser.purs

```diff
@@ -0,0 +1,100 @@
+module Test.E2E.Browser
```

Comment:
I think all of the `Test.E2E.*` modules should be moved to a different part of the module hierarchy. Many (most?) of our users will want to use e2e testing, but the current module naming/location suggests its for CTL's internal tests

Perhaps we can have a `Contract.Test.*` namespace, with `E2E` modules therein